lib/librpmi_internal.h

```diff
@@ -34,20 +34,33 @@
 #define offsetof(TYPE, MEMBER) ((rpmi_size_t) &((TYPE *)0)->MEMBER)
 #endif
 
+#undef container_of
```

Comment:
Instead of `#undef` use `#ifndef` here and everywhere below.

Reply:
We should not rely on host firmware for these because we will have no control over how these macros will be implemented by host fw

Reply:
in which case, add "rpmi_" prefix to these macro names and drop the `#undef`

Reply:
some macros here may have common understanding but others like CLAMP, ROUNDUP, ROUNDDOWN, there is none

Reply:
adding "rpmi_" prefix does'nt change the understanding of macros.

Reply:
that comment was before yours updated :)

Reply:
done